src/main/resources/logback.xml

```diff
@@ -19,18 +19,6 @@
           <pattern>${ISO8601_DATE_FORMAT}</pattern>
           <fieldName>created</fieldName>
         </timestamp>
-        <Pattern>
```

Comment:
are these not used then?

Reply:
Yeah, the span and thread were used by zipkin.  The service says 'sample' which it isnt.  This'll remove 3 fields that were just taking up space and providing no value